dts/bindings/sensor/ti,ina3221.yaml

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Texas Instruments INA3221 Triple-Channel Current/Power Monitor
+
+compatible: "ti,ina3221"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  conv-time-shunt:
+    type: int
+    default: 4
+    description: |
+      Shunt-voltage conversion time.
+      0 = 140 μs
+      1 = 204 μs
+      2 = 332 μs
+      3 = 588 μs
+      4 = 1.1 ms (default)
+      5 = 2.116 ms
+      6 = 4.156 ms
+      7 = 8.244 ms
+    enum:
+      - 0
+      - 1
+      - 2
+      - 3
+      - 4
+      - 5
+      - 6
+      - 7
+  conv-time-bus:
+    type: int
+    default: 4
```

Comment:
The default value needs a justification in the description. Is it a register reset value?

Reply:
same as above